templates.json

```diff
@@ -206,6 +206,15 @@
         "repo": "https://github.com/openfaas-incubator/python-flask-template",
         "official": "true"
     },
+    {
+        "template": "python3-flask-debian",
+        "platform": "x86_64",
+        "language": "Python",
+        "source": "openfaas-incubator",
+        "description": "Python 3.6 Flask template based on Debian",
```

Comment:
3.6 or 3.7?

Reply:
Oh, 3.7.  I just copy/pasted from what was there for the others.  Which are 3.7 anyway.  So all of those need to be updated.